soc/arm/ti_k3/am62x_m4/soc.h

```diff
@@ -0,0 +1,5 @@
+/*
+ * Copyright (c) 2023 L Lakshmanan
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
```

Comment:
If we are going to leave this file blank do we really need it?
What happens if you remove `#include <soc.h>` from `soc.c` ?

Reply:
I have kept the file for future additions to the soc files. As of now though, it builds fine without `soc.h`.